src/utils/firebase.ts

```diff
@@ -119,22 +120,24 @@ const getAllDocsFromCollection = async (collectionName: string) => {
     return mapQuerySnapshotToDocs(querySnapshot);
 };
 
-const getLocationDict = async (collectionName: string) => {
-    const docs = await getAllDocsFromCollection(collectionName);
+const getPackingInputsDict = async () => {
+    const docs = await getAllDocsFromCollection(FIRESTORE_COLLECTIONS.PACKING_INPUTS);
     // docs is an array of objects, each with an id, a name, and other fields
     // we want to create a dictionary with the name as the key and the original_location as the value
     // `reduce` is a method that takes an array and reduces it to a single value
-    const locationDict = docs.reduce((locationDict: FirebaseDict, doc: FirestoreDoc) => {
+    const inputsDict: Dictionary<PackingInputs> = docs.reduce((inputsDict: Dictionary<PackingInputs>, doc: FirestoreDoc) => {
         const name = doc[FIRESTORE_FIELDS.NAME];
-        const id = doc.id;
-        if (name) {
-            locationDict[name] = {
-                [FIRESTORE_FIELDS.FIREBASE_ID]: id,
+        const config = doc[FIRESTORE_FIELDS.CONFIG];
+        const recipe = doc[FIRESTORE_FIELDS.RECIPE];
+        if (name && config && recipe) {
+            inputsDict[name] = {
+                [FIRESTORE_FIELDS.CONFIG]: config,
+                [FIRESTORE_FIELDS.RECIPE]: recipe,
             };
-        } 
-        return locationDict;
-    }, {} as FirebaseDict);
-    return locationDict;
+        }
+        return inputsDict;
+    }, {} as Dictionary<PackingInputs>);
```

Comment:
nit: just realized we don't need this `as Dictionary<PackingInputs>` cast here anymore because the return type is already defined.